flink-connectors/flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/util/HBaseTestingClusterAutoStarter.java

```diff
@@ -119,6 +123,9 @@ private static void initialize(Configuration c) {
 
 	@BeforeClass
 	public static void setUp() throws Exception {
+		// HBase 2.2.3 HBaseTestingUtility works with only a certain range of hadoop versions
+		String hadoopVersion = System.getProperty("hadoop.version");
+		Assume.assumeTrue(HADOOP_VERSION_RANGE.contains(hadoopVersion));
```

Comment:
There is a utility org.apache.hadoop.util.VersionUtil provided in hadoop for comparing version. What about making use of it?

Reply:
Range is using VersionUtil::compareVersions now to compare.